scripts/exregional_get_extrn_mdl_files.sh

```diff
@@ -69,12 +69,26 @@ if [ "${ICS_OR_LBCS}" = "ICS" ]; then
 
 elif [ "${ICS_OR_LBCS}" = "LBCS" ]; then
   file_set="fcst"
-  first_time=$((TIME_OFFSET_HRS + LBC_SPEC_INTVL_HRS))
   if [ "${FCST_LEN_HRS}" = "-1" ]; then
     CYCLE_IDX=$(( ${cyc} / ${INCR_CYCL_FREQ} ))
     FCST_LEN_HRS=${FCST_LEN_CYCL[$CYCLE_IDX]}
   fi
-  last_time=$((TIME_OFFSET_HRS + FCST_LEN_HRS))
+  end_hr=$FCST_LEN_HRS
+  if [ $BOUNDARY_LEN_HRS -gt $end_hr ]; then
+     end_hr=$BOUNDARY_LEN_HRS
+  fi
+  # There is a difference between RRFS_dev & SRW regarding download of 0th
+  # hour lbcs. RRFS_dev downloads 0th hour lbcs while SRW does not.
+  # I tried to make them the same but RRFS_dev workflow skips make_ics for
+  # some cycles so it needs to download 0th hour lbcs.
```

Comment:
Please read this comment in case we have a better way to handle this difference.

Reply:
I have made it so that NEED_ALL_LBCS is passed from the j-job. It is currently turned on for RRFS workflow but we could go a step further and put the flag in config_defaults.yaml.